src/modules/createChannel/CreateChannel.tsx

```diff
@@ -1,128 +1,340 @@
 // This is the Parent component for the Create Channel flow.
+// React + web3 essentials
+import { useEffect, useState } from 'react';
+
+// Third party libraries
+import { useFormik } from 'formik';
+import { ethers } from 'ethers';
```

Comment:
Create channel button should be disabled when we are added token from testnet so user cannot initiate the chanel creation process when he is not allowed to.